content/java/concepts/strings/terms/getChars/getChars.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.getChars()'
+Description: 'Copies characters from the given string into the destination character array.'
+Subjects:
+  - 'Java'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **.getChars()** string method copies a set of characters from a string into a destination character array. 
+
+## Syntax
+
+The basic syntax of `.getChars()` in Java is:
+
+```pseudo
+getChars(int srcBegin, int srcEnd, char[] dst, int dstBegin)
+```
+A brief description of the parameters is as follows:
+
+- `srcBegin`: The index to start copying the new array.
+- `scrEnd`: The index to stop copying the new array.
+
+dst: - 'The name of the new array. '
+dstBegin: The index of the destination array.
+
+## Example
+
+In the following example, `getChar()` converts an array into a new shorter array. It then prints the new array in the console 
+
+```class java
+class example{ 
+  public static void main(Strings args [])
+  {
+    string str = "Welcome to Texas";
+    char [] destArray = new chars[5];
+    try {
+      str.getChars(11, 16, destArray, 0)
+      System.out.println(newArray);
+    }
+     catch (Exception ex) {
+      System.out.println("The string is too short.");
+    }
+    
+  }
```

Comment:
```suggestion
class example{ 
  public static void main(String args [])
  {
    String str = "Welcome to Texas";
    char[] destArray = new char[5];
    try {
      str.getChars(11, 16, destArray, 0)
      System.out.println(destArray);
    }
     catch (Exception ex) {
      System.out.println("The string is too short.");
    }  
  }
```

Reply:
I think this exception message is a little off. If the indexes in the getchars method are out of bounds it will result in an error, so, I think it might be more accurate to use a message such as-

The passed indexes are out of range.

*Addtionally, for future notice, I've noticed that this example is very close to an example on another site. Ideally, examples are created for docs and are 100% original.